index.html

```diff
@@ -323,6 +323,253 @@ <h2 class="h2">opacity</h2>
       <div class="bg-cyan-500 text-white opacity-50">opacity 50</div>
       <div class="bg-cyan-500 text-white opacity-70">opacity 70</div>
       <div class="bg-cyan-500 text-white opacity-90">opacity 90</div>
+    </div> -->
+
+    <!-- Cards -->
+    <!-- <div class="test-2">
+      <div class="flex shadow rounded-md h-68">
+        <div class="badge-container img-container">
+          <img
+            class="rounded-md img-responsive"
+            src="https://picsum.photos/350/250"
+            alt=""
+            srcset=""
+          />
+          <span class="absolute card-badge badge bg-rose-500 text-white"
+            >New</span
+          >
+        </div>
+        <div class="card-description p-6">
+          <h5 class="h5 font-bold">Title</h5>
+          <div class="flex mt-3 mb-28">
+            <p class="subtitle1 line-through mr-3">Price</p>
+            <p class="subtitle1 text-rose-500">price</p>
+          </div>
+          <div class="flex">
+            <a
+              href="#"
+              class="btn bg-rose-600 bg-hover-rose-400 text-white mr-4"
+              ><span><i class="btn-icon fas fa-shopping-cart"></i></span>Add to
+              Cart</a
+            ><a href="#" class="btn btn-link-slate"
+              ><span><i class="btn-icon fas fa-shopping-cart"></i></span>Add to
+              wishlist</a
+            >
+          </div>
+        </div>
+      </div>
+    </div> -->
+
+    <!-- <div class="test-2 w-80">
+      <div class="flex flex-col shadow-md rounded-md">
+        <div class="badge-container img-container">
+          <img
+            class="rounded-md img-responsive"
+            src="https://picsum.photos/350/250"
+            alt=""
+            srcset=""
+          />
+          <span class="absolute card-badge badge bg-rose-500 text-white"
+            >New</span
+          >
+        </div>
+        <div class="flex flex-col p-5">
+          <div class="flex justify-between align-items-center">
+            <h5 class="h5 font-semibold">Title</h5>
+            <i class="fas fa-heart text-grey-500 text-hover-rose-500"></i>
+          </div>
+          <div class="flex mt-3 mb-10">
+            <p class="subtitle1 line-through mr-3">Price</p>
+            <p class="subtitle1 text-rose-500">price</p>
+          </div>
+
+          <a class="btn bg-rose-600 bg-hover-rose-400 w-36 text-white"
+            ><span><i class="btn-icon fas fa-shopping-cart"></i></span>Add to
+            Cart</a
+          >
+        </div>
+      </div>
+    </div>
+    <div class="test-2 w-80">
+      <div class="flex flex-col shadow-md rounded-md">
+        <div class="badge-container img-container">
+          <img
+            class="rounded-md img-responsive"
+            src="https://picsum.photos/350/250"
+            alt=""
+            srcset="https://picsum.photos/350/250"
```

Comment:
Suggestions:
src and srcset are relating to the same image link so you can keep either one of them no need to keep both
Put a value in the alt attribute

Reply:
okay will make the changes